scripts/copy-contract-artifacts.ts

```diff
@@ -0,0 +1,18 @@
+import * as fs from "fs"
+
+const contractArtifactsPath = "apps/contracts/build/contracts/contracts/Feedback.sol"
+
+const subgraphArtifactsPath = "apps/subgraph/contract-artifacts"
+
+const webAppArtifactsPath = "apps/web-app/contract-artifacts"
+
+async function copyArtifactsFiles() {
```

Comment:
```suggestion
async function main() {
   try {
        await fs.promises.copyFile(`${contractArtifactsPath}/Feedback.json`, `${subgraphArtifactsPath}/Feedback.json`)
        await fs.promises.copyFile(`${contractArtifactsPath}/Feedback.json`, `${webAppArtifactsPath}/Feedback.json`)
    } catch (err) {
        throw new Error("Error occurred while copying the contract artifacts files")
    }
}

main()
    .then(() => process.exit(0))
    .catch((error) => {
        console.error(error)
        process.exit(1)
    })
```

What about using the Hardhat standard we already used here? https://github.com/semaphore-protocol/semaphore/blob/main/scripts/download-snark-artifacts.ts